README.md

```diff
@@ -35,6 +35,7 @@
 - [Using Auth0 with Static Web Apps](https://www.aaron-powell.com/posts/2021-05-13-using-auth0-with-static-web-apps/) [www.aaron-powell.com]
 - [Introducing the Static Web Apps CLI](https://techcommunity.microsoft.com/t5/apps-on-azure/introducing-the-azure-static-web-apps-cli/ba-p/2257581) [techcommunity.microsoft.com]
 - [Use Azure Static Web Apps with Azure DevOps pipelines](https://www.daveabrock.com/2021/04/01/static-web-apps-azure-pipelines/) [www.daveabrock.com]
+- [Microsoft Releases Azure Static Web Apps into General Availability](https://www.infoq.com/news/2021/05/azure-static-web-apps-ga/) [www.infoq.com]
```

Comment:
I'm not sure that this article really fits this list, it's more a third-party write up of the announcement than bringing new learning material to the list.

Reply:
Oops! I got confused, I was doing 10 different things in parallel and I thought I had removed that link 😅